templates/TIP.html.tmpl

```diff
@@ -209,6 +208,14 @@
                 <td class="content-wrappable">Git Repo Component <em>{{id}}</em></td>
                 <td class="content-wrappable">{{data.git.url}}</td>
             </tr>
+            <tr>
+                <td class="content-wrappable">ODS Component Type <em>{{id}}</em></td>
+                <td class="content-wrappable">{{type}}</td>
+            </tr>
+            <tr>
+                <td class="content-wrappable">Installation by ODS <em>{{id}}</em></td>
+                <td class="content-wrappable">{{doInstall}}</td>
```

Comment:
We could add a new Attribute to Metadata.yml eg. Installinstruction ... and in case of saas add it here in brackets